kex-runner/src/main/kotlin/org/vorpal/research/kex/reanimator/actionsequence/ActionSequence.kt

```diff
@@ -187,6 +188,61 @@ class ReflectionList(
     }
 }
 
+class MockSequence(
+    name: String,
+    val mockCalls: MutableList<MockCall>, // create instance and setup methods
+    val reflectionCalls: MutableList<ReflectionCall> // only setup fields
```

Comment:
What do you mean by 'setup fields' here? What is the purpose of these reflection calls?

Reply:
And why is it explicitly a mutable list of `ReflectionCall` and not just `ReflectionList` or `ActionSequence`?